application/src/main/java/org/togetherjava/tjbot/commands/search/GoogleSearchStrategy.java

```diff
@@ -0,0 +1,28 @@
+package org.togetherjava.tjbot.commands.search;
+
+import io.mikael.urlbuilder.UrlBuilder;
+
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.concurrent.CompletableFuture;
+
+public class GoogleSearchStrategy extends SearchStrategy {
+    private static final String API_KEY =
+            "1d84500b082341aad66de5cf02f7c83d8608411b01b7ffcbb2d40042eea250df";
```

Comment:
that has to move into config. id suggest u invalidate this token. people can now use it to do harm in ur name.

Reply:
That's a test token I generated using dummy credentials. I can change this once you have created that resource file you mentioned earlier.